metricbeat/module/docker/diskio/_meta/fields.yml

```diff
@@ -4,15 +4,69 @@
     Disk I/O metrics.
   release: ga
   fields:
+    - name: read
+      type: group
+      description: >
+        Accumulated reads during the life of the container
+      fields:
+        - name: ops
+          type: long
+          description: >
+            Number of reads during the life of the container
+        - name: bytes
+          type: long
```

Comment:
Could you add here `format: bytes` to kibana can make sense of it. Same is true for other bytes values.

Reply:
Done